### PR TITLE
Fixed controller extension name longer than 64 chars error not showing up

### PR DESCRIPTION
**Pull request by Dannysht**
Bug fix for #1419 

- Added proper functionality to display error when name is over 64 characters long

### CODE REVIEW

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
* please add a changeset - should be a patch for `preview-middleware` since the client is shipped with the middleware
* please also sign the CLA
* otherwise, coding is clean and properly tested

**Comment by nikmace** [comment]
Please update the code and resolve the conflicts. Link to the changed method: [controller](https://github.com/SAP/open-ux-tools/blob/main/packages/preview-middleware-client/src/adp/controllers/ControllerExtension.controller.ts#L65C48-L66C1)

File: .changeset/stale-papayas-do.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+---
+
+Fixed controller extension name longer than 64 chars error not showing up
```
Comment (zdravko-georgiev) [inline .changeset/stale-papayas-do.md:5]: Please add also the `@sap-ux/preview-middleware`, `patch` is sufficient.

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]

File: .changeset/stale-papayas-do.md (hunk 1 above)
Comment (Dannysht) [reply .changeset/stale-papayas-do.md]: so like this?
---
'@sap-ux-private/preview-middleware-client': patch
'@sap-ux/preview-middleware': patch
---

Fixed controller extension name longer than 64 chars error not showing up

**Review by zdravko-georgiev — COMMENTED** [review]
@Dannysht Do we need this check also for the `fragment` names? If yes, could you please also include it in this PR?

File: .changeset/stale-papayas-do.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/preview-middleware': patch
+---
+
+Fixed controller extension name longer than 64 chars error not showing up
```
Comment (zdravko-georgiev) [inline .changeset/stale-papayas-do.md:6]: Please update this message 😉

**Review by zdravko-georgiev — CHANGES_REQUESTED** [review]

File: .changeset/stale-papayas-do.md (hunk 2 above)
Comment (zdravko-georgiev) [reply .changeset/stale-papayas-do.md]: thx

**Review by zdravko-georgiev — APPROVED** [review]
Change is small and easy to understand
Excellent test coverage
Review comments addressed
Manual testing ✅

**Review by nikmace — APPROVED** [review]
The code looks good.
Coverage is good.
Did not test manually.

**Review by tobiasqueck — APPROVED** [review]
Requested changes have been applied

**Pull request merged**